Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -253,15 +253,16 @@ class TabManagerImplementation: LegacyTabManager, Notifiable {
                                          tabHistoryCurrentState: state)
 
             let tabId = UUID(uuidString: tab.tabUUID) ?? UUID()
-            if tab.url?.absoluteString == nil {
+            if tab.url == nil && tab.lastKnownUrl == nil {
```

Comment:
I'm simply confused here, not by the code itself but by what exactly you are trying to achieve here.
If both are nil then you want to log it but still proceed with the below TabData instance.

Wouldn't it be better to capture and log which URL is nil 🤔 

Unless I am missing something 😅

Reply:
I just quickly modified the log I had before. The idea would be that if both are nil, then we have a problem (so logging it). If only one of them is nil, then it's not an issue. I could modify it to log which one is nil, would be more value that way I guess